.github/workflows/scripts/validate-coderabbit.sh

```
@@ -0,0 +1,44 @@
+echo "Step 1: Fetching all PR reviews with pagination..."
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)